### PR TITLE
convert timestamp to iso9601

### PR DESCRIPTION
**Pull request by knockknockyoo**

### CODE REVIEW

File: services/user.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
-const {getCurrentTimeYYYYMMDDSS} = require("../utility/time-utility");
+
 const {v4} = require("uuid")
 const {USER} = require("../constants/user-constants");
 const {ERROR} = require("../constants/error-constants");
 const {UpdateProfileEvent} = require("../domain/log-events");
+const {getCurrentTimeYYYYMMDDSS, toISO} = require("../utility/time-utility");
```
Comment (n2iw) [inline services/user.js:6]: getCurrentTimeYYYYMMDDSS function should be renamed to something like getCurrentDateTime.

File: services/user.js [hunk 2]
```diff
@@ -16,15 +17,9 @@ class User {
...
-        if (!params?.userID) {
-            throw new Error(ERROR.INVALID_USERID);
-        };
-        if (context?.userInfo?.role !== USER.ROLES.ADMIN && context?.userInfo.role !== USER.ROLES.ORG_OWNER) {
-            throw new Error(ERROR.INVALID_ROLE);
-        };
-        if (context?.userInfo?.role === USER.ROLES.ORG_OWNER && !context?.userInfo?.organization?.orgID) {
-            throw new Error(ERROR.NO_ORG_ASSIGNED);
-        }
+        if (!params?.userID) throw new Error(ERROR.INVALID_USERID);
```
Comment (n2iw) [inline services/user.js:20]: Please avoid using one-liner if statement, keep the **{}** even it's one line.

File: services/user.js [hunk 3]
```diff
@@ -144,6 +139,11 @@ class User {
     }
 }
 
+const toISOTime = (aUser) => {
```
Comment (n2iw) [inline services/user.js:142]: This function name is a bit misleading, people may think we are converting the object into an ISO time. I suggest rename it to something like transformDateTime, transformDateTimeFormat or similar.

**Review by n2iw — CHANGES_REQUESTED** [review]
Please see detailed comment in separate comments.

File: services/user.js (hunk 3 above)
Comment (knockknockyoo) [reply services/user.js]: updated!

File: services/user.js (hunk 2 above)
Comment (knockknockyoo) [reply services/user.js]: updated!

File: services/user.js (hunk 1 above)
Comment (knockknockyoo) [reply services/user.js]: updated!

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**